migrate_pim/upgrade/upgrade_from_32_to_40.rst

```diff
@@ -179,6 +179,14 @@ new Asset Manager feature: https://github.com/akeneo/CsvToAsset
 Migrating your custom code
 **************************
 
+About parameters.yml
```

Comment:
should we keep these docs 3_2 to 4_0?

Reply:
There is a page about incremental documentation:
https://github.com/akeneo/pim-docs/blob/master/migrate_pim/upgrade_major_version.rst
I did not date to break this logic event though i agree with you upgrade could be run incrementally following each upgrade documentation.